Net31Solution/SensorApp/SensorApp.csproj

```diff
@@ -14,4 +14,8 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Xml" Version="6.0.0" />
```

Comment:
Add UI folder and put the code files related to ui into it. 
Ensure that code files are distributed between Data/Business/UI folders.